examples/notebooks/rag/requirements.txt

```diff
@@ -1,5 +1,8 @@
 ## Data prep kit
 
+data-prep-toolkit==0.2.1
```

Comment:
data-prep-toolkit and data-prep-toolkit-ray should not be needed as the other wheels depend on these 2.

Reply:
David, the idea is to "enforce" the dependencies to the lowest possible level at the application level.